tests/monday_types/time_zone_type_tests.py

```diff
@@ -0,0 +1,109 @@
+import json
+import pytz
+from pytz.exceptions import UnknownTimeZoneError
+from schematics.exceptions import ValidationError,DataError
+from nose.tools import eq_,raises
+from unittest.mock import patch
+
+from moncli import entities as en
+from moncli import types as t
+from moncli import client
+from moncli.enums import ColumnType
+from moncli.models import MondayModel
+
+
+@patch.object(en.Item,'get_column_values')
```

Comment:
You do not need any of these mocked calls.  

All you need to do is create a column value (in isolation of an item), create a timezone_type instance, pass the column_value into the to_native method, and verify the returned value.